server/controllers/assess/assessments/pagesController.test.ts

```diff
@@ -18,10 +19,12 @@ import paths from '../../../paths/assess'
 import { viewPath } from '../../../form-pages/utils'
 
 import { assessmentFactory, clarificationNoteFactory } from '../../../testutils/factories'
+import { remapArsonAssessmentData } from '../../../form-pages/utils/matchingInformationUtils'
 
 const PageConstructor = jest.fn()
 
 jest.mock('../../../utils/validation')
+jest.mock('../../../form-pages/utils/matchingInformationUtils')
```

Comment:
Can we use `jest.spyOn()` on the relevant method we're expecting to have been called instead of mocking the whole module? Mocking the whole module affects the rest of the test suite, and does not return actual implementation which makes it difficult to track other affected logic when/if some of the implementation itself changes.

Reply:
Make sense